### PR TITLE
Add Russian translation for Linux .desktop file

### PR DESCRIPTION
**Pull request by OlesyaGerasimenko**

### CODE REVIEW

**Comment by AThousandShips** [comment]
Thank you for your contribution

However this is not where these files are edited, instead you edit them on the webplate, see [here](https://docs.godotengine.org/en/stable/contributing/documentation/editor_and_docs_localization.html)

Closing as this is not a valid change

**Comment by AThousandShips** [comment]
My bad was wrong area

**Review by akien-mga — APPROVED** [review]
Checked with Google Translate, seems good.

**Comment by JekSun97** [comment]
As a native speaker I can say that everything is correct

**Review by H3XAGON3ST — APPROVED** [review]
I confirm that the translation is correct

File: misc/dist/linux/org.godotengine.Godot.desktop [hunk 1]
```diff
@@ -4,11 +4,13 @@ GenericName=Libre game engine
 GenericName[el]=Ελεύθερη μηχανή παιχνιδιού
 GenericName[fr]=Moteur de jeu libre
 GenericName[nl]=Libre game-engine
+GenericName[ru]=Свободный игровой движок
 GenericName[zh_CN]=自由的游戏引擎
 Comment=Multi-platform 2D and 3D game engine with a feature-rich editor
 Comment[el]=2D και 3D μηχανή παιχνιδιού πολλαπλών πλατφορμών με επεξεργαστή πλούσιο σε χαρακτηριστικά
 Comment[fr]=Moteur de jeu 2D et 3D multiplateforme avec un éditeur riche en fonctionnalités
 Comment[nl]=Multi-platform 2D- en 3d-game-engine met een veelzijdige editor
```
Comment (dalexeev) [inline misc/dist/linux/org.godotengine.Godot.desktop:12]:
```suggestion
Comment[nl]=Multi-platform 2D- en 3D-game-engine met een veelzijdige editor
```

I don't know Dutch, but it looks like an obvious typo.

**Review by dalexeev — APPROVED** [review]
The translation and spelling are correct.

**Comment by AThousandShips** [comment]
Please squash your commits into one, see [here](https://docs.godotengine.org/en/latest/contributing/workflow/pr_workflow.html#the-interactive-rebase)

**Comment by OlesyaGerasimenko** [comment]
> Please squash your commits into one, see [here](https://docs.godotengine.org/en/latest/contributing/workflow/pr_workflow.html#the-interactive-rebase)

Can I somehow do it in the web interface?

**Comment by AThousandShips** [comment]
No you have to use the command line

**Comment by akien-mga** [comment]
I can squash it myself if you're not familiar with that workflow.

**Comment by OlesyaGerasimenko** [comment]
> I can squash it myself if you're not familiar with that workflow.

That would be great, thank you!

**Comment by akien-mga** [comment]
Thanks!

**Pull request merged**